omero/sysadmins/index.rst

```diff
@@ -36,22 +38,28 @@ series of commands.
     :titlesonly:
 
     unix/server-installation
+    unix/install-web/web-deployment
+    unix/server-binary-repository
+    unix/server-postgresql
+    advanced-install
+
+.. toctree::
```

Comment:
Can you move this toctree on to the unix/server-installation page to match what you've done for web? I think the previous and next navigation should make sense once that is done